GRIB2_CodeFlag_4_2_CodeTable_en.csv

```diff
@@ -1065,7 +1078,8 @@ Parameter number by product discipline and parameter category,"Product disciplin
 Parameter number by product discipline and parameter category,"Product discipline 10 - Oceanographic products, parameter category 191: miscellaneous",255,,Missing,,,Operational
 Parameter number by product discipline and parameter category,"Product discipline 20 - Health and socioeconomic impacts, parameter category 0: health indicators",0,,Universal thermal climate index,,K,Operational
 Parameter number by product discipline and parameter category,"Product discipline 20 - Health and socioeconomic impacts, parameter category 0: health indicators",1,,Mean radiant temperature,,K,Operational
-Parameter number by product discipline and parameter category,"Product discipline 20 - Health and socioeconomic impacts, parameter category 0: health indicators",2-191,,Reserved,,,Operational
+Parameter number by product discipline and parameter category,"Product discipline 20 - Health and socioeconomic impacts, parameter category 0: health indicators",2,,Wet-bulb globe temperature,"(see Note 1)",K,Operational
```

Comment:
@amilan17 Note_en is (see Note 1). There are other Note 1 in this table. Be careful to have the right order in doc/pdf. How do we manage the duplicated Note number?

Reply:
@chenxiaoxia2019 - in table 4.2, each unique combination of discipline/category has notes that are numbered for that section. In other words, the note numbers are not sequentially applicable to the entire code table, they are specific to each combination of discipline and category.